app/assets/javascripts/application.js

```diff
@@ -34,4 +34,6 @@
 //= require modernizr
 
 //= require orangelight
+//= require availability
```

Comment:
It didn't work without these explicit requires. Maybe there's a setting that in the config that we don't have enabled?